Task 1.4/Task 1.4/main.c

```diff
@@ -0,0 +1,26 @@
+#include <stdio.h>
+
+int main(int argc, const char * argv[]) {
+    int array[19] = {0};
+    
+    
+    for (int i = 0; i <=9; i++) {
+        for (int j = 0 ; j <= 9; j++) {
+            for (int q = 0; q<=9; q++) {
```

Comment:
```suggestion
            for (int q = 0; q <= 9; q++) {
```
С пробелами надо аккуратнее